features/provider/finance/invoices.feature

```diff
@@ -0,0 +1,43 @@
+@ignore-backend @javascript
+Feature: Show all provider's invoices
+  In order to manage invoices
+  As a provider
+  I want to be able to see all invoices issued by me
+
+  Scenario: Display years of invoices when there are invoices
+    Given a provider is logged in on 1st January 2011
+    And a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
+    And the provider is charging its buyers in prepaid mode
+    And an invoice of buyer "zoidberg" for January, 2011
+    And an invoice of buyer "zoidberg" for January, 2014 on 1st January 2014 (without scheduled jobs)
+    When I go to the invoices issued by me
```

Comment:
We talked about not using `I...` and `me` steps anymore in new cucumbers. I suggest simply adding a new option in paths.rb:
https://github.com/3scale/porta/blob/f2df57d44306cefb9aac9aea92ae78e35a3c46a4/features/support/paths.rb#L587

Since the provider (not me) is the one logged in, I would say "the provider/admin is at the invoices page".

Reply:
@josemigallas if we are not having a "cucumber 101" session any time soon, can you please just pop all of your "cucumber guidelines" in a Google doc or something, so we can discuss/agree/use it going forward?

I know that it's the same comments in every PR, but it is very hard to remember such things, if they mainly just sit in your head :wink:

Reply:
> I suggest simply adding a new option in paths.rb:

Hmm... not sure what you mean. The option in `paths.rb` was already there, that's why I just used the standard "I go to" approach. Or do you mean to change the expression for this path?

Reply:
I made some related changes in https://github.com/3scale/porta/pull/3485/commits/922896c5d2fc5b49308c006d1b9d8939a858c035
What do you think?